shared/js/companies.js

```diff
@@ -118,7 +121,7 @@ var Companies = (() => {
                 // uncomment for testing
                 //storageData.twitter = {count: 10, name: 'twitter', pagesSeenOn: 10}
                 storageData = Companies.sanitizeData(storageData)
-                for (company in storageData) {
+                for (let company in storageData) {
```

Comment:
Another undefined variable that was causing problems.

There isn't a `company` (lower case) variable anywhere else in the background code, so I just scoped it to this `for` loop.